content/en/account_management/teams.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Teams
+kind: documentation
+---
+
+## Overview
+Datadog Teams allow groups of users to organize their team assets within Datadog and automatically filter their Datadog-wide experience to prioritize these assets.
+
+Use Teams to link resources such as dashboards, services, monitors, and incidents to a group of users. You can also add team-specific links to Slack channels, Jira boards, GitHub repositories, etc.
+
+Team membership is flexible. Users can join teams or be added by an administrator. Users can belong to multiple teams. 
+
+## Setup
+
+### Navigation
+
+To find the [team directory page][1]:
+
+1. Navigate to **Organization Settings** from your account menu.
+1. Under **Groups**, select **Teams**.
+
+The [Team directory page][1] lists all teams within your organization. Use the **Only My Teams** slider in the upper right to toggle the view between your teams and all teams in the organization.
+
+{{< img src="account_management/teams-directory.png" alt="Organization Settings page, Teams tab, showing a panel highlighting the Caching team" >}}
+
+### Create team
+
+1. On the [Team directory page][1], click **New Team** at the upper right.
+1. Choose a **Team Name**.
+1. The **Handle** populates based on your team name.
+1. Optionally, choose an **Avatar Color**.
+1. Provide an optional **Summary**.
+1. Use the drop-down to select team members.
+1. Click **Create**.
+
+### Modify team
+
+1. On the [Team directory page][1], click the team you would like to modify. A side panel appears with the team details.
+1. Mouse over the item you wish to modify. A pencil icon appears.
+1. Click the pencil icon. A pop-up window appears.
+1. Make your changes, then click the appropriate button to save your changes.
+
+## Team handle
+
+A team handle links teams to Datadog resources. Team handles appear in search bars and facets in the format `team:<team-name>`. 
```

Comment:
Two things:
1. It should be `team:<team-handle>` and not `team:<team-name>`
2. For the incidents product, the facet is actually `teams` and not `team`. Not sure if that is worth pointing out.